integration-tests/basic/src/main/java/Basic.java

```diff
@@ -1,4 +1,4 @@
-public class IntegrationTest {
+public class Basic {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [All classes and interfaces must belong to a named package](https://www.codacy.com/app/sgoertzen/sonar-break-maven-plugin/file/2300516131/issues/source?bid=3070326&fileBranchId=3194199#l1)